make install fails on cargo install

error logs:

```shell
❯ make -j $(nproc) install
[PROBE] building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/core/probe/kernel/bpf ...
[PROBE] building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/core/probe/user/bpf ...
[HOOK]  building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/module/ct/bpf ...
[HOOK]  building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/module/nft/bpf ...
[HOOK]  building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/module/ovs/bpf ...
[HOOK]  building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/module/skb/bpf ...
[HOOK]  building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/module/skb_drop/bpf ...
[HOOK]  building /home/pvalerio/workspace/open_source/retis-org/retis/retis/src/module/skb_tracking/bpf ...
[CARGO] building retis (release) ...
error: found a virtual manifest at `/home/pvalerio/workspace/open_source/retis-org/retis/Cargo.toml` instead of a package manifest
make: *** [Makefile:63: install] Error 101
```

--- a/Makefile
+++ b/Makefile
@@ -60,7 +60,7 @@ EBPF_HOOKS := $(abspath $(wildcard retis/src/module/*/bpf))
 all: debug
 
 install: release
-	$(CARGO) $(CARGO_OPTS) install $(CARGO_INSTALL_OPTS) --path=. --offline --frozen
+	$(CARGO) $(CARGO_OPTS) install $(CARGO_INSTALL_OPTS) --path=$(ROOT_DIR)/retis --offline --frozen
 
 define build
 	$(call out_console,CARGO,$(strip $(2)) ...)

